fedbadges/rules.py

```diff
@@ -187,13 +187,10 @@ def _get_candidates(self, msg: Message, tahrir: TahrirDatabase):
             candidates = frozenset([krb2fas(uri) for uri in candidates])
 
         # Skipped usernames
-        for candidates in self.config.get("skip_users"):
-            if candidates == self.config.get("skip_users"):
-                candidates = frozenset(candidates - self.config.get("skip_users"))
-            else:
-                continue
+        candidates = candidates.difference(frozenset(self.config.get("skip_users", [])))
 
         # Remove None
+
```

Comment:
You don't really need to add this empty line, comments are usually right above what they are commenting.